migrate-3-to-4/mongo.sh

```diff
@@ -5,11 +5,11 @@ function import_mongo() {
 
     MONGO_POD="mongodb-0"
     MONGODB_USERNAME="root"
-    MONGODB_PASSWORD=$(kubectl -n "$NAMESPACE" get secrets mongodb -o jsonpath="{.data.mongodb-root-password}" | base64 --decode)
+    MONGODB_PASSWORD=$(kubectl -n "$NAMESPACE" get secrets mongodb-credentials -o jsonpath="{.data.mongodb-root-password}" | base64 --decode)
```

Comment:
```suggestion
    MONGODB_PASSWORD=$(kubectl -n "$NAMESPACE" get secrets mongodb -o jsonpath="{.data.mongodb-root-password}" | base64 --decode)
```

Mongo secret name is `mongo` only. thanks @ryanwohara